spectrum-scanner/tools/README.md

```diff
@@ -0,0 +1,40 @@
+# RIOT Spectrum Scanner Application
+
+## About
+
+This script can be used to plot the radio signal spectrum when a connected node
+is running the [spectrum-scanner application](https://github.com/RIOT-OS/applications/tree/master/spectrum-scanner)
```

Comment:
Now that script is part of this repo and within the scanner-app directory, this section can be shortened a bit, e.g. removing the links?